src/test/java/bio/terra/service/snapshot/flight/delete/DeleteSnapshotDeleteSamGroupStepTest.java

```diff
@@ -27,36 +25,33 @@
 @Tag(Unit.TAG)
 class DeleteSnapshotDeleteSamGroupStepTest {
   @Mock IamService iamService;
+  @Mock SnapshotRequestDao snapshotRequestDao;
   private UUID snapshotId;
   @Mock private FlightContext flightContext;
   private DeleteSnapshotDeleteSamGroupStep step;
+  private static final String expectedName = "samGroupName";
+  private static final SnapshotAccessRequestModel snapshotAccessRequestModel =
```

Comment:
We're not always strict about this in test classes, but static final objects (except for `logger`) should be all caps
```suggestion
  private static final String EXPECTED_NAME = "samGroupName";
  private static final SnapshotAccessRequestModel REQUEST_MODEL =
```